### ISSUE TITLE
"404 This game is not published and can no longer be downloaded" when redeeming key

### ISSUE THREAD

**Issue by gh89275** [itchio/itch.io#858]
I just tried to redeem a key for Collisions by 2DEngine and was told

> 404 Not Found
> 
> This project is not available for download
> 
> This game is not published and can no longer be downloaded. If you think this is a mistake, please get in touch with support.

I do think this is a mistake - a policy mistake by itch. For comparison, Steam has a policy of not screwing over paid-up customers. If a game is in your library it will always stay there and be playable, and if a key is redeemable it will always be redeemable (until redeemed) - even if the game is withdrawn from sale or the publisher banned from the platform. (maybe an exception would be made for actual malware, but I've never heard of it on Steam)

**Comment by leafo** [comment]
Can you email us at support@itch.io with your download link if you haven't already. We have warnings in place to notify developers that they might be restricting access to customers but a bug may not have notified them.

We can pull the developer in on the email thread and figure out what happened and why the page was deactivated.

Thanks

**Comment by gh89275** [comment]
Thanks for replying. When you say "download link", is that the itch key that has not yet been linked to my account?

Also, might this issue also apply to games that were successfully linked to my account in the past, but might have disappeared without me noticing?

**Comment by leafo** [comment]
yeah, the download link is the thing that's 404ing (the link sent in your email receipt). It's needed so support can look up the purchase.

**Comment by leafo** [comment]
> Also, might this issue also apply to games that were successfully linked to my account in the past, but might have disappeared without me noticing?

There are two ways a developer can disable a page:

* Outright delete it. It will not show up anywhere on the site any more. We block this for sellers if someone owns the game. They need to go through support if they need to get it deleted
* Put the page in draft. We don't block this, but we warn the developer if they'll be disabling access to people. The game will still show up in your library (iirc) but clicking on a link will take you to 404

**Comment by gh89275** [comment]
> We block this for sellers if someone owns the game.

What if nobody bought it direct through itch, but got itch keys from a groupees bundle? This is my situation with Collisions and also, I discovered, yy slider by nekomuragames. These were keys I tried to redeem yesterday. I haven't checked all the keys I previously redeemed to make sure the games still exist!

I'll email you the keys.

**Comment by gh89275** [comment]
> I haven't checked all the keys I previously redeemed

I started checking my library. I've got less than half way and there are 11 browser tabs with "404" in them. I'd prefer to focus on the games I bought in bundles, that the devs have received money for (and I received an itch key). Is there an easy way to distinguish them from the games that were free to "claim" at some point?

**Comment by gh89275** [comment]
> Can you email us at support@itch.io with your download link if you haven't already

I emailed you 8 days ago (subject "404 when trying to redeem keys"), with no reply. I used the same email that my itch account uses, which has a hyphen in the username and ends with @yahoo.com.au - if that makes any difference. To add to that email: one of the keys I complained about actually *was* once redeemed, but has since gone 404, as have the games in the previous comment.
